### PR TITLE
doc: Explain GitHub action PAT

### PR DESCRIPTION
**Pull request by DafyddLlyr**

### CODE REVIEW

File: .github/workflows/cron-deploy-to-production.yml [hunk 1]
```diff
@@ -14,8 +14,10 @@ jobs:
     steps:
     - name: sync
       env:
-        # Uses `PROD_DEPLOY_PAT` which is a Personal Access Token generated by a planx-new repo admin
+        # Requires a Personal Access Token generated by an OSL GitHub org admin
         # Allows this action to trigger the push-production.yml action
+        # PAT requires the "Read and write" permissions on repository "Contents"
+        # XXX: Expires 01/12/23, see docs here on generating a new one - https://bit.ly/3YNr5sM
```
Comment (jessicamcinchak) [inline .github/workflows/cron-deploy-to-production.yml:20]: maybe it's worth scheduling a slack reminder or gcal event for this expiration date?

**Review by jessicamcinchak — APPROVED** [review]
thanks for docs!

File: .github/workflows/cron-deploy-to-production.yml (hunk 1 above)
Comment (DafyddLlyr) [reply .github/workflows/cron-deploy-to-production.yml:20]: Done! 👍

Comment (gunar) [reply .github/workflows/cron-deploy-to-production.yml:20]: Did you create this PAT under your own account @DafyddLlyr? I was considering creating a brand new GitHub account just for this but perhaps this is fine for now.

Comment (gunar) [reply .github/workflows/cron-deploy-to-production.yml:20]: Maybe instead of a PAT under someone's account we'd be better off with a Deploy Key.

**Pull request merged**